contracts/modules/STO/CappedSTO.sol

```diff
@@ -178,9 +161,9 @@ contract CappedSTO is ISTO, ReentrancyGuard {
             endTime,
             cap,
             rate,
-            fundsRaised,
+            (fundRaiseType[uint8(FundRaiseType.POLY)]) ? fundsRaisedPOLY: fundsRaisedPOLY,
```

Comment:
This looks wrong if -else is returning same value

Reply:
resolved